src/group_theory/quotient_group.lean

```diff
@@ -266,6 +266,29 @@ def equiv_quotient_of_eq {M N : subgroup G} [M.normal] [N.normal] (h : M = N) :
   right_inv := λ x, x.induction_on' $ by { intro, refl },
   map_mul' := λ x y, by rw map_mul }
 
+/-- Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →* B / (B' ⊓ B)` induced by the inclusions. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`. If `A' ≤ B'` and `A ≤ B`,
+then there is a map `A / (A' ⊓ A) →+ B / (B' ⊓ B)` induced by the inclusions."]
+def quotient_map_of_subgroups_le {A' A B' B : subgroup G}
+  [hAN : (A'.subgroup_of A).normal] [hBN : (B'.subgroup_of B).normal]
+  (h' : A' ≤ B') (h : A ≤ B) :
+  quotient (A'.subgroup_of A) →* quotient (B'.subgroup_of B) :=
+map _ _ (subgroup.inclusion h) $
+  by simp [subgroup.subgroup_of, subgroup.comap_comap]; exact subgroup.comap_mono h'
+
+/-- Let `A', A, B', B` be subgroups of `G`.
+If `A' = B'` and `A = B`, then the quotients `A / (A' ⊓ A)` and `B / (B' ⊓ B)` are isomorphic. -/
+@[to_additive "Let `A', A, B', B` be subgroups of `G`.
+If `A' = B'` and `A = B`, then the quotients `A / (A' ⊓ A)` and `B / (B' ⊓ B)` are isomorphic."]
+def equiv_quotient_of_subgroups_eq {A' A B' B : subgroup G}
+  [hAN : (A'.subgroup_of A).normal] [hBN : (B'.subgroup_of B).normal]
+  (h' : A' = B') (h : A = B) :
+  quotient (A'.subgroup_of A) ≃* quotient (B'.subgroup_of B) :=
```

Comment:
Is this more general statement true?
```suggestion
def equiv_quotient_of_equiv {A : subgroup G} {B : subgroup H'}
  [hAN : A.normal] [hBN : B.normal]
  (h : A ≃* B) :
  quotient (A'.subgroup_of A) ≃* quotient (B'.subgroup_of B) :=
```
which you could then pair with a proof that `A'.subgroup_of A ≃* B'.subgroup_of B`

Reply:
Ok lol I just spent 30 minutes trying to prove this and I suddenly realized its not true.

Take `Z_2 x Z_4`. You can find subgroups of size 2 which give quotients of `Z_2 x Z_2` and `Z_4`